invenio_stats/templates.py

```diff
@@ -29,5 +29,7 @@
 
 def register_templates():
     """Register elasticsearch templates for events."""
-    return ['contrib/{0}'.format(event['event_type'])
-            for event in current_stats._events_config.values()]
+    event_templates = ['contrib/{0}'.format(event['event_type'])
+                       for event in current_stats._events_config.values()]
+    aggregation_templates = ['contrib/aggregations']
```

Comment:
The aggregations should be added only if the corresponding events have also been added. We will probably have N events -> M aggregations, where M>N, because multiple aggregations can be deduced from the same event. Thus we could just have an `_aggregations_config` and `aggregations` in `current_stats` which would work the same way as events. That way we let the user choose which aggregations he wants.